aws/resources/route53_hostedzone.go

```diff
@@ -62,9 +64,14 @@ func (r *Route53HostedZone) nukeRecordSet(id *string) (err error) {
 		return err
 	}
 
+	// get the domain name
+	var domainName = awsgo.StringValue(r.HostedZonesDomains[awsgo.StringValue(id)].Name)
```

Comment:
Hm, interesting can be the case that `r.HostedZonesDomains[key]` will not match any value?

Reply:
I don't think there's a case where r.HostedZonesDomains[key] won't match any value. In the code, we only keep the resources that are selected for nuke, and Route 53 always has a domain name.